test/tests.jl

```diff
@@ -185,6 +185,21 @@ end
                 @test occursin("\nh(x::Int32, y::Int32, z::Int32; kwargs...) -> Int32\n", str)
             end
             @test occursin("\n```\n", str)
+
+            doc.data = Dict(
+                :binding => Docs.Binding(M, :h),
+                :typesig => Tuple{Int},
+                :module => M,
+            )
+            DSE.format(DSE.TYPEDSIGNATURES, buf, doc)
+            str = String(take!(buf))
+            @test occursin("\n```julia\n", str)
+            if typeof(1) === Int64
+                @test occursin("\nh(x::Int64) -> Int64\n", str)
+            else
+                @test occursin("\nh(x::Int32) -> Int32\n", str)
+            end
```

Comment:
For some reason, the docstring is the following if the typesig is `Tuple{Int, Int, Int}`:

```
h(x::Int64, y::Int64, z::Int64; kwargs...) -> Int64
```

but is the following if the typesig is `Tuple{Int}`:

```
h(x::Int64) -> Int64
```

It's missing the `; kwargs...`. 

I'm not sure how to get that to print correctly.